modules/nodes-secondary-scheduler-configuring-console.adoc

```diff
@@ -0,0 +1,72 @@
+
+:_content-type: PROCEDURE
+[id="nodes-secondary-scheduler-configuring-console_{context}"]
+= Deploying a secondary scheduler
+
+After you have installed the {secondary-scheduler-operator}, you can deploy a secondary scheduler.
+
+.Prerequisities
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+* The {secondary-scheduler-operator-full} is installed.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+. Create config map to hold the configuration for the secondary scheduler.
+.. Navigate to *Workloads* -> *ConfigMaps*.
+.. Click *Create ConfigMap*.
+.. In the YAML editor, enter the config map definition that contains the necessary `KubeSchedulerConfiguration` configuration:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "secondary-scheduler-config"                  <1>
+  namespace: "openshift-secondary-scheduler-operator" <2>
+data:
+  "config.yaml": |
+    apiVersion: kubescheduler.config.k8s.io/v1beta1
+    kind: KubeSchedulerConfiguration                  <3>
+    leaderElection:
+      leaderElect: false
+    profiles:
+      - schedulerName: secondary-scheduler            <4>
+        plugins:                                      <5>
+          score:
+            disabled:
+              - name: NodeResourcesBalancedAllocation
+              - name: NodeResourcesLeastAllocated
+----
+<1> The name of the config map. This is used in the *Scheduler Config* field when creating the `SecondaryScheduler` CR.
+<2> The config map must be created in the `openshift-secondary-scheduler-operator` namespace.
+<3> The `KubeSchedulerConfiguration` resource for the secondary scheduler. For more information, see link:https://kubernetes.io/docs/reference/config-api/kube-scheduler-config.v1beta2/#kubescheduler-config-k8s-io-v1beta2-KubeSchedulerConfiguration[`KubeSchedulerConfiguration`] in the Kubernetes API documentation.
+// TODO: This link is v1beta2 though and our example is using v1beta1...
+<4> The name of the secondary scheduler. Pods that set their `spec.schedulerName` field to this value are scheduled with this secondary scheduler.
+<5> The plug-ins to enable or disable for the secondary scheduler. For a list scheduling plug-ins, see link:https://kubernetes.io/docs/reference/scheduling/config/#scheduling-plugins[Scheduling plugins] in the Kubernetes documentation.
++
+// TODO: Is this an okay list to link to? Are there others to call out - or should we say nothing?
+
+.. Click *Create*.
+
+. Create the `SecondaryScheduler` CR:
+.. Navigate to *Operators* -> *Installed Operators*.
+.. Select *{secondary-scheduler-operator-full}*.
+.. Select the *Secondary Scheduler* tab and click *Create SecondaryScheduler*.
+.. The *Name* field defaults to `cluster`; do not change this name.
++
+// TODO: should the UI prevent you from changing this if this one is required?
```

Comment:
@ingvagabund if the CR name MUST be "cluster", should the UI prevent users from changing it or anything? Or have a warning or note in the UI (if that's even possible)

Reply:
Open question. The same holds for the descheduler and other operators which expect a hard-coded name for their configuration object. Let's keep this as it is. The UI is written for an arbitrary component. Forcing a specific name or printing a warning would be difficult. @jhadvig or maybe not?